contracts/LockedTokens.cdc

```diff
@@ -166,6 +173,7 @@ pub contract LockedTokens {
         /// Called by the admin every time a vesting release happens
         pub fun increaseUnlockLimit(delta: UFix64) {  
             self.unlockLimit = self.unlockLimit + delta
+            emit UnlockLimitIncreased(address: self.owner!.address, increaseAmount: delta, newLimit: self.unlockLimit)
```

Comment:
Just checking: Is the resource not having an owner an unrecoverable error?
If not, consider using optional chaining instead.

The same applies to the two lines below which emit an event with the address of the resource owner